react-components/src/communities/styles.js

```diff
@@ -33,9 +33,14 @@ const styles = theme => ({
         height: "100%",
     },
 
+    table: {
+        height: "calc(100% - " + theme.mixins.toolbar['@media (min-width:600px)'].minHeight + "px)",
+        overflow: "auto",
+    },
+
```

Comment:
😲

Reply:
lol that was real frustrating to figure out, but once I got it 😌